### PR TITLE
Expand comm epic doc

### PR DESCRIPTION
**Pull request by willingc**
In preparation for #4902

### CODE REVIEW

File: packages/epics/docs/comms.md [hunk 1]
```diff
@@ -1,15 +1,34 @@
...
+Comm epics, more formally Communication Epics, are epics that support interacting with the
+[Jupyter Messaging Protocol](https://jupyter-client.readthedocs.io/en/stable/messaging.html)
+via comm messages.
 
-A comm 
+A few years ago, Jupyter added a
+[custom messaging](https://jupyter-client.readthedocs.io/en/stable/messaging.html#custom-messages)
+system for developers to add their own objects with Frontend and Kernel-side components,
+and allow them to communicate with each other. To do this, IPython adds a notion of a `Comm`,
+which exists on both sides (Frontend and Kernerl), and can communicate in either direction.
```
Comment (captainsafia) [inline packages/epics/docs/comms.md]:
```suggestion
which exists on both sides (Frontend and Kernel), and can communicate in either direction.
```

Comment (captainsafia) [inline packages/epics/docs/comms.md]: Nit: Let's use "front end" for the noun and "front-end" for the adjective per [the content guide](https://content-guide.nteract.io/#/word-list).

**Review by captainsafia — APPROVED** [review]
Looks good overall! I left some comments and suggestions inline.

File: packages/epics/docs/comms.md (hunk 1 above)
Comment (willingc) [reply packages/epics/docs/comms.md]: That's so cool that there is a content guide.

**Pull request merged**